### DISCUSSION TITLE
Field "register" argument "name" of type "String!" is required but not provided

### DISCUSSION THREAD

**Discussion by metal3d** [TabbyML/tabby#2264] [General]
**Describe the bug**
After compiling myself to ensure Cuda 12.4.1 is used, I started the server and see this:

![image](https://github.com/TabbyML/tabby/assets/100080/8eb844d3-8291-4894-a380-45d0316b2f57)

Field "register" argument "name" of type "String!" is required but not provided

**Information about your version**
main branch on May the 28th

**Comment by wsxiaoys** [comment]
ui assets hasn't been updated in main branch yet, so the issue is WAI. if you're interested in creating a customized build - consider reusing an released tag, e.g `v0.11.1`

**Comment by metal3d** [comment]
OK, I understand.

Now the problem is essentially my issue on #2266 ;)

Thanks for your reply.
